2_pvc_destination_gen/extras/container/stunnel.conf

```diff
@@ -4,9 +4,8 @@ socket = l:TCP_NODELAY=1
 socket = r:TCP_NODELAY=1
 sslVersion = TLSv1.2
 
-[ssh]
-TIMEOUTclose = 0
```

Comment:
Any consequences of removing `TIMEOUTclose` ?

Reply:
I dont know, I thought that was a `[ssh]` module-specific, I was not sure if it is available in `[rsync]`

Reply:
fixed it @pranavgaikwad